### PR TITLE
LPS-115543 Use UnmodifiableList where is possible.

### PR DESCRIPTION
**Pull request by cgoncas**

### CODE REVIEW

File: modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/model/AssetVocabularyMetric.java [hunk 1]
```diff
@@ -82,6 +95,10 @@ public int hashCode() {
 		return Objects.hash(_assetCategoryMetrics);
 	}
 
+	public boolean isEmpty() {
```
Comment (cgoncas) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/model/AssetVocabularyMetric.java:98]: Very confusing that the method isEmpty return another thing other than this instance != _EMPTY.

File: modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/provider/AssetVocabulariesProvider.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+import java.util.stream.Stream;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author David Arques
+ */
+@Component(service = AssetVocabulariesProvider.class)
+public class AssetVocabulariesProvider {
```
Comment (cgoncas) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/provider/AssetVocabulariesProvider.java:35]: We should provide a test for this component

Comment (cgoncas) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/provider/AssetVocabulariesProvider.java:35]: Maybe in the next pull?

File: modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/provider/AssetVocabulariesProvider.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+	public List<AssetVocabulary> getAssetVocabularies(long companyId) {
+		Group companyGroup = _groupLocalService.fetchCompanyGroup(companyId);
```
Comment (cgoncas) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/provider/AssetVocabulariesProvider.java:38]: As we don't have another group, call companyGroup group, is just fine.

**Comment by darquesdev** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/model/AssetVocabularyMetric.java (hunk 1 above)
Comment (darquesdev) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/model/AssetVocabularyMetric.java:98]: true! changing it ;)

File: modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/provider/AssetVocabulariesProvider.java (hunk 2 above)
Comment (darquesdev) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/provider/AssetVocabulariesProvider.java:35]: yes, in the next PR I'll provide a unit test

File: modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/model/AssetVocabularyMetric.java (hunk 1 above)
Comment (darquesdev) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/model/AssetVocabularyMetric.java:98]: I just got rid of it ;)

**Comment by darquesdev** [comment]
Pull request submitted to https://github.com/darquesdev/liferay-portal/pull/176. [See changes here.](https://github.com/darquesdev/liferay-portal/compare/2de3e936ac4c~1...2de3e936ac4c) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**